tests/framework/helpers/JsonTest.php

```diff
@@ -33,7 +33,7 @@ public function testEncode()
         $dataArrayable = $this->getMockBuilder('yii\\base\\Arrayable')->getMock();
         $dataArrayable->method('toArray')->willReturn([]);
         $actual = Json::encode($dataArrayable);
-        $this->assertSame('{}', $actual);
+        $this->assertSame('[]', $actual);
```

Comment:
I think it makes sense that Arrayable object would return array not dictionary. This test was wrong.